File: java-apps/kq-common/src/main/java/com/kafkaquest/kq/common/util/models/events/responsedata/GameStartedEventResponseData.java
package com.kafkaquest.kq.common.util.models.events.responsedata;

import com.kafkaquest.kq.common.util.models.events.other.JigsawPuzzlePieceResponse;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@EqualsAndHashCode(callSuper = false)
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class GameStartedEventResponseData extends ResponseData {
    List<JigsawPuzzlePieceResponse> puzzlePieces;
}

File: kq-projection-dotnet/kq-projection/Common/Models/Events/GameStartedEventResponseData.cs
using KQ_Projection.Common.Models.Jigsaw;
using System.Text.Json.Serialization;

namespace KQ_Projection.Common.Models.Events
{
    public class GameStartedEventResponseData : ResponseData
    {
        [JsonPropertyName("puzzlePieces")]
        public List<JigsawPuzzlePieceResponse> puzzlePieces { get; set; }
    }
}
